metadata-service/plugin/src/main/java/com/datahub/plugins/configuration/Config.java

```diff
@@ -0,0 +1,44 @@
+package com.datahub.plugins.configuration;
+
+import com.datahub.plugins.common.Validator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nonnull;
+import lombok.Builder;
+import lombok.Getter;
+
+
+/**
+ * {@link Config} is getting loaded from /etc/datahub/plugins/auth/config.yaml
+ */
+@Getter
+@Builder
+@JsonDeserialize(builder = Config.CustomBuilder.class)
+public class Config {
```

Comment:
Needs test

Reply:
The first test in TestPluginFramework is for this Config

Reply:
Added test for testing ConfigProvider where Config is covered (TestConfigProvider)

Reply:
Added TestConfig